src/components/BorrowerProfile/CommentsAndWhySpecial.vue

```diff
@@ -278,13 +343,34 @@ export default {
 				this.isLightboxVisible = false;
 				this.loading = false;
 			});
-		}
+		},
+		randomizedUserClass() {
+			const randomStyle = this.userCardStyleOptions[Math.floor(Math.random() * this.userCardStyleOptions.length)];
+			return `${randomStyle.color} ${randomStyle.bg}`;
+		},
+		determineIfMobile() {
+			this.isMobile = document.documentElement.clientWidth < 735;
+		},
+		isDefaultProfilePic(commentHash) {
```

Comment:
Logic taken from header to determine if it's the placeholder avatar

Reply:
Might be good in a shared util, though not a huge priority, since it's a small method with what I'm guessing are IDs that don't ever change.

Reply:
Yeah the ids shouldn't change.